integration/spark-common-test/pom.xml

```diff
@@ -116,7 +116,6 @@
   </dependencies>
 
   <build>
-    <testSourceDirectory>src/test/scala</testSourceDirectory>
```

Comment:
Is it required to remove this?

Reply:
I have removed this because I have added java file . And that java file also need to be compiled.